datatrails-common-api/assets/v2/assets/merklelogentry.proto

```diff
@@ -8,12 +8,32 @@ import "google/protobuf/timestamp.proto";
 // MerkeLogCommit provides the log entry details for a single mmr leaf.
 message MerkleLogCommit {
     /* The mmr index */
-    uint64 index = 3;
+    uint64 index = 1;
     /* The mmr *leaf* index */
-    uint64 leaf_index = 4; // TBD: this may be redundant.
+    uint64 leaf_index = 2; // TBD: this may be redundant.
     /* time ordered and strictly unique per tenant. system wide
      * unique with very reasonable operational assumptions. */
-    fixed64 idtimestamp = 5;
+    fixed64 idtimestamp = 3;
+}
+
+message MerkleLogCommitMongoDB {
```

Comment:
I am not a big fan of calling this MongoDB, this is implementation detail exposed to our customers?

I am happy for `idtimestamp` to be a hexstring, we will just have to ensure our tooling and libraries are away of the fact.

Reply:
We need a different message for this to work, the type name isn't exposed in the json the customer sees. Can you propose a name you prefer ?

Reply:
I think we resolved this in chat